README.md

```diff
@@ -125,7 +127,21 @@ LOGARDIAN_LABELS=http,database # output log with http and database labels
 LOGARDIAN_LABELS=emails,* # output every log with label
 ```
 
-  
+#### I do not want to see caller and path. How can I turn off them globally?
+
+Specify 'false' on LOGARDIAN_TRACE in *.env*. Works only in dev. 
```

Comment:
Please specify that it's default behavior in prod env. From what you wrote now it seems that it won't work in prod at all and available only while dev env is used